cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterHealthService.java

```diff
@@ -54,12 +68,19 @@ public class ClouderaManagerClusterHealthService implements ClusterHealthService
 
     static final String HOST_AGENT_CERTIFICATE_EXPIRY = "HOST_AGENT_CERTIFICATE_EXPIRY";
 
+    // TODO Do we extend this to include additional health check params?
```

Comment:
Convert to a jira and add to the comment, or remove the comment.

Reply:
Updated